classes/order/OrderReturn.php

```diff
@@ -74,20 +81,17 @@ public function addReturnDetail($order_detail_list, $product_qty_list, $customiz
                 }
             }
         }
-        /* Customized product return */
-        if ($customization_ids) {
-            foreach ($customization_ids as $order_detail_id => $customizations) {
-                foreach ($customizations as $customization_id) {
-                    if ($quantity = (int) $customization_qty_input[(int) $customization_id]) {
-                        Db::getInstance()->insert('order_return_detail', ['id_order_return' => (int) $this->id, 'id_order_detail' => (int) $order_detail_id, 'product_quantity' => $quantity, 'id_customization' => (int) $customization_id]);
-                    }
-                }
-            }
-        }
     }
 
-    public function checkEnoughProduct($order_detail_list, $product_qty_list, $customization_ids, $customization_qty_input)
+    public function checkEnoughProduct($order_detail_list, $product_qty_list)
     {
+        if (func_num_args() > 2) {
+            @trigger_error(
```

Comment:
Same here:
```suggestion
    public function checkEnoughProduct($order_detail_list, $product_qty_list, $customization_ids = null, $customization_qty_input = null)
    {
        if ($customization_ids !== null || $customization_qty_input !== null) {
            @trigger_error(
```

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/ba5f7ae6de13acbf71a11cd67c5a7e0e9cd912b4